### ISSUE TITLE
Oban Cron: provide a way to set owner of monitor

### ISSUE THREAD

**Issue by Miradorn** [getsentry/sentry-elixir#824]
Hi there and thanks for the great library!

I'm wondering wether it would be possible to provide a way to set the `owner` of a cron monitor via the automatic integration?

We have a setup where we know the owning Sentry team in our codebase and if we could somehow tell the integration to set it correctly it would ease our debugging flow a lot.

e.g. by specifying a callback or some kind of mapping. You probably know best how to achieve it! 

Thanks ❤

**Comment by sl0thentr0py** [comment]
@savhappy @solnic can one of you add the `owner` field to check ins and expose it in the api too?

**Comment by savhappy** [comment]
@sl0thentr0py yep, I have some time later today.

**Comment by whatyouhide** [comment]
This is now possible through the changes in #833.

**Comment by Miradorn** [comment]
@whatyouhide as it turns out, since the options of checking are validated with NimpleOptions it actually doesn't work right now. I guess the specs of the Checkin needs to be updated. I might be able to prepare a PR later for it!

**Comment by whatyouhide** [comment]
Yep we need to add this to that spec. Thanks in advance for the PR 🤟
